### PR TITLE
[PE-4579] Updated form component styles for better consistency with Portal and Brand

### PR DESCRIPTION
**Pull request by mattnickles**
- Changed form focus color value from Chi Blue 60 to Chi Blue 50.
- Updated all button and form components with consistent focus state.
- Changed Checkbox and Radio Button border with from 2px to 1px for better consistency with Portal and Brand.
- Changed Secondary and Light button text color from Chi's default text color to black.
- Removed legacy code from Button styles.
- Updated copy in Button documentation examples.
- Added missing -text class to text in Tab documentation examples.

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://8a900564cf7f2829573912c8137474f50cda9dad-chi-pr.enterprise.ctl.io" target="_blank"> https://8a900564cf7f2829573912c8137474f50cda9dad-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/819/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/820/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://773d13d9d2efa22ca079ebe317644612bab06ac2-chi-pr.enterprise.ctl.io" target="_blank"> https://773d13d9d2efa22ca079ebe317644612bab06ac2-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/821/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://99404bbcb584acb301d195d2e0e32aacae3b6a95-chi-pr.enterprise.ctl.io" target="_blank"> https://99404bbcb584acb301d195d2e0e32aacae3b6a95-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/823/ for more details.

File: src/chi/components/range-slider/range-slider.scss [hunk 1]
```diff
@@ -160,7 +118,7 @@ input {
       &::-webkit-slider-thumb {
         @include thumb-style;
         -webkit-appearance: none;
-        margin-top: -6px;
+        margin-top: -8px;
```
Comment (jllr) [inline src/chi/components/range-slider/range-slider.scss]: @mattnickles can these px measures be converted to rems?

File: src/chi/components/buttons/buttons.scss [hunk 2]
```diff
@@ -484,46 +475,24 @@ $sizes: (
...
-            color: $text-color;
+          .chi-icon {
+            color: $text-black;
           }
 
+          &:active,
+          &.-active,
           &:hover,
-          &.-hover {
+          &.-hover, {
```
Comment (jllr) [inline src/chi/components/buttons/buttons.scss]: @mattnickles is this comma necessary?

Comment (mattnickles) [reply src/chi/components/buttons/buttons.scss]: The entire `-icon` rule wasn't necessary as we no longer specify custom styles for flat light icon buttons. Removed - thanks!

File: src/chi/components/range-slider/range-slider.scss (hunk 1 above)
Comment (mattnickles) [reply src/chi/components/range-slider/range-slider.scss]: Changed them all. Thanks!

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://1b5eaf3f17baac766d616b9d723c7d6dd7524e7c-chi-pr.enterprise.ctl.io" target="_blank"> https://1b5eaf3f17baac766d616b9d723c7d6dd7524e7c-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/826/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://068199bf2d7c52966a752644f7217c6493573c44-chi-pr.enterprise.ctl.io" target="_blank"> https://068199bf2d7c52966a752644f7217c6493573c44-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/827/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://923f4365c6dfa9fc718e8a6dcbb9538abe98e74d-chi-pr.enterprise.ctl.io" target="_blank"> https://923f4365c6dfa9fc718e8a6dcbb9538abe98e74d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/828/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-3i6v02, https://ops.elasticbox.com/services/instances/i-sie2o2, https://ops.elasticbox.com/services/instances/i-9x4iok, https://ops.elasticbox.com/services/instances/i-typ8cw, https://ops.elasticbox.com/services/instances/i-cu8m17, https://ops.elasticbox.com/services/instances/i-no7ve9, https://ops.elasticbox.com/services/instances/i-2gct13

**Pull request merged**